src/features/comments/commentsSlice.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
```

Comment:
It is recommended to avoid using 'eslint-disable' unless it's absolutely necessary. Consider fixing the code to satisfy the ESLint rule instead of disabling it.